### ISSUE TITLE
Broken link at Volume/iSCSI example, issue related to githubbranch param 

### ISSUE THREAD

**Issue by ClaudiaJKang** [kubernetes/website#29439]
**This is a Bug Report**

**Problem:**

On [concept/storage document](https://kubernetes.io/docs/concepts/storage/volumes/#iscsi), the iSCSI example link is not working properly on 1.21 version. Because the link is broken. When clicking the iSCSI example, the document of the latest version redirects the url([https://github.com/kubernetes/examples/tree/main/volumes/iscsi
](https://github.com/kubernetes/examples/tree/main/volumes/iscsi)). And the document of release-1.21 version redirects the url([https://github.com/kubernetes/examples/tree/release-1.21/volumes/iscsi](https://github.com/kubernetes/examples/tree/release-1.21/volumes/iscsi)).

---
![Screen Shot 2021-08-15 at 14 03 57](https://user-images.githubusercontent.com/17492826/129467728-865a7236-270a-4f81-9736-ac67824f9ce2.png)
----

![Screen Shot 2021-08-15 at 14 04 04](https://user-images.githubusercontent.com/17492826/129467732-5d6e08dd-5d4f-469a-9441-82701b882f39.png)
----
![Screen Shot 2021-08-15 at 13 52 16](https://user-images.githubusercontent.com/17492826/129467543-5963b73b-6c40-4471-9d1b-aa0cd96f673f.png)
----

**Proposed Solution:**

`the parameter "githubbranch"` of the original document([docs/concepts/storage/volumes.md](https://github.com/kubernetes/website/blob/main/content/en/docs/concepts/storage/volumes.md
)) occur this problem because the kubernetes/examples repository doesn't have `main` and `release-1.21` branch.

- [docs/concepts/storage/volumes.md](https://github.com/kubernetes/website/blob/main/content/en/docs/concepts/storage/volumes.md
)
```md
See the [iSCSI example](https://github.com/kubernetes/examples/tree/{{< param "githubbranch" >}}/volumes/iscsi) for more details.
```

This issue has been solved partially by #29311.(This PR change `{{<param "githubbranch"}}` to `master`) 
But, I think that the "githubbranch" parameter is changed according to the documentation branch in order to support minor version documentation.

**Page to Update:**
https://kubernetes.io/docs/concepts/storage/volumes/#iscsi

**Comment by sftim** [comment]
As a _bug_ I think this was fully fixed in PR #29311 

The examples repo doesn't have tags per release that we can hyperlink to.

**Comment by sftim** [comment]
@ClaudiaJKang you might be requesting a backport of the fix from #29311 to the v1.21 docs

If that's what you want to happen, I recommend opening an issue framed around that backport request.
